src/sentry/grouping/fingerprinting/configs/sentry.javascript@2024-02-02/001_chunkload_errors.txt

```diff
@@ -0,0 +1,2 @@
+family:javascript error.type:"ChunkLoadError" -> chunkloaderror
+family:javascript error.value:"ChunkLoadError*" -> chunkloaderror
```

Comment:
Have we tested if `family:javascript error.type:"ChunkLoadError" error.value:"ChunkLoadError*" -> chunkloaderror` would do the trick? (combining both rules)